modules/custom/reservations/reservations.routing.yml

```diff
@@ -0,0 +1,7 @@
+reservations.content:
+  path: '/reservations'
```

Comment:
Možda da dodas da je ruta 'movie-reservations:, da ruta sama po sebi opise šta se rezervise. Jednog dana ako dodas knjige bilo bi book-reservations, it'd.